### PR TITLE
docs: explicit boolean operators

### PR DESCRIPTION
**Pull request by rvantonder**
https://sourcegraph.slack.com/archives/C0W2E592M/p1620058895327000

### CODE REVIEW

**Comment by camdencheek** [comment]
Looks like there are a [few places](https://sourcegraph.com/search?q=context:global+repo:github%5C.com/sourcegraph/sourcegraph%24+%23operators&patternType=literal) this anchor is used for URLs

**Comment by emchap** [comment]
An elegant solution! :)

**Review by emchap — APPROVED** [review]

**Review by emchap — APPROVED** [review]

**Comment by rvantonder** [comment]
@camdencheek

File: doc/code_search/how-to/opengrok.md [hunk 1]
```diff
@@ -51,9 +51,9 @@ The only exceptions are colon characters, which are by default used for specifyi
 
 Oracle OpenGrok provides three boolean operators — `AND`, `OR`, and `NOT` — for scoping searches to files that contain strings that match multiple patterns.
 
-Sourcegraph also provides [`AND`, `OR`, and `NOT` operators](../reference/queries.md#operators).
+Sourcegraph also provides [`AND`, `OR`, and `NOT` operators](../reference/queries.md#boolean-operators).
 
-> NOTE: Operators are available as of Sourcegraph 3.15 and enabled with `{"experimentalFeatures": {"andOrQuery": "enabled"}}` in the site configuration. Built-in operator support is planned for an upcoming release.
```
Comment (rvantonder) [inline doc/code_search/how-to/opengrok.md:56]: holy potatoes

**Pull request merged**